[Bug][Resource LogDrain] The `type` property seems to be required

While trying to create a Log Drain, I realized that the `type` property is in fact required by the API but by the Terraform provider.

> Note: `type` is mandatory. Mandatory attributes will depend on the chosen type, please refer to the first table.

(Source: https://developers.scalingo.com/log_drains)

This is confirmed as the following code throws an error:

```tf
resource "scalingo_app" "test" {
  name = "test-app"
}

resource "scalingo_log_drain" "elk" {
  app = scalingo_app.test.id
  url = var.logstash_uri
}
```

```shell
$ terraform apply
...
Error: fail to create log drain: fail to add drain: 400 Bad Request → 'type' parameter is required

  with scalingo_log_drain.elk,
  on test.tf line 4, in resource "scalingo_log_drain" "elk":
  34: resource "scalingo_log_drain" "elk" {

Error: Terraform exited with code 1.
Error: Process completed with exit code 1.
```

The `type` attribute must therefore be changed to make it mandatory.

--- a/scalingo/resource_scalingo_log_drain.go
+++ b/scalingo/resource_scalingo_log_drain.go
@@ -26,7 +26,7 @@ func resourceScalingoLogDrain() *schema.Resource {
 			"type": {
 				Type:        schema.TypeString,
 				ForceNew:    true,
-				Optional:    false,
+				Required:    true,
 				Description: "Type of log drain (syslog/http/elk/etc.)",
 			},
 			"host": {

--- a/scalingo/resource_scalingo_log_drain.go
+++ b/scalingo/resource_scalingo_log_drain.go
@@ -27,7 +27,7 @@ func resourceScalingoLogDrain() *schema.Resource {
 				Type:        schema.TypeString,
 				ForceNew:    true,
 				Required:    true,
-				Description: "Type of log drain (syslog/http/elk/etc.)",
+				Description: "type of log drain (syslog/http/elk/etc.)",
 			},
 			"host": {
 				Type:        schema.TypeString,

--- a/scalingo/resource_scalingo_log_drain.go
+++ b/scalingo/resource_scalingo_log_drain.go
@@ -26,7 +26,7 @@ func resourceScalingoLogDrain() *schema.Resource {
 			"type": {
 				Type:        schema.TypeString,
 				ForceNew:    true,
-				Optional:    true,
+				Optional:    false,
 				Description: "Type of log drain (syslog/http/elk/etc.)",
 			},
 			"host": {

